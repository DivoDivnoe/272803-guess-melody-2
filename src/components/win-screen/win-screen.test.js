import React from 'react';
import renderer from 'react-test-renderer';
import WinScreen from './win-screen.jsx';

describe(`WinScreen component`, () => {
  it(`is rendered correctly`, () => {
    const gameTime = 30;
    const mistakes = 1;
    const points = 10;
    const renderButton = jest.fn();

    const tree = renderer.create(
        <WinScreen
          gameTime={gameTime}
          mistakes={mistakes}
          points={points}
          renderButton={renderButton}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
