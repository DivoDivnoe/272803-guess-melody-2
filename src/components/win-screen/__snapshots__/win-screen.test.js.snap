// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WinScreen component is rendered correctly 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="result__title"
  >
    Вы настоящий меломан!
  </h2>
  <p
    className="result__total"
  >
    За 0 минут и 30 секунд{getTimeEnding(seconds)} вы набрали 12 баллов (8 быстрых), совершив 1 ошибки
  </p>
  <p
    className="result__text"
  >
    Вы заняли 2 место из 10. Это лучше чем у 80% игроков
  </p>
</section>
`;
